models/image_segmentation_efficientsam/demo.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import numpy as np
+import cv2 as cv
+from efficientSAM import EfficientSam
+
+# Check OpenCV version
+assert cv.__version__ >= "4.9.0", \
+       "Please install latest opencv-python to try this demo: python3 -m pip install --upgrade opencv-python"
```

Comment:
The OpenCV version should be `>= 4.10.0`

Reply:
Friendly reminder